app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<h1>Dashboard</h1>
+<table border="1">
+  <thead>
+    <th>Tenant</th>
+    <th>API Key</th>
+    <th>API Request Count</th>
+  </thead>
+  <tbody>
+    <% @tenants.each do |tenant|%>
+      <tr>
+        <td><%= tenant.name%></td>
+        <td><%= tenant.api_key %></td>
```

Comment:
Yes i know is security risk, i just kept for you guys to use the api key to check Tenant api request with Curl, otherwise i dont keep sensitive data in frontend.